netty/src/main/java/io/grpc/netty/Utils.java

```diff
@@ -62,18 +63,20 @@
  */
 class Utils {
 
-  public static final ByteString STATUS_OK = new ByteString("200".getBytes(UTF_8));
-  public static final ByteString HTTP_METHOD = new ByteString(GrpcUtil.HTTP_METHOD.getBytes(UTF_8));
-  public static final ByteString HTTPS = new ByteString("https".getBytes(UTF_8));
-  public static final ByteString HTTP = new ByteString("http".getBytes(UTF_8));
-  public static final ByteString CONTENT_TYPE_HEADER =
-      new ByteString(CONTENT_TYPE_KEY.name().getBytes(UTF_8));
-  public static final ByteString CONTENT_TYPE_GRPC =
-      new ByteString(GrpcUtil.CONTENT_TYPE_GRPC.getBytes(UTF_8));
-  public static final ByteString TE_HEADER = new ByteString("te".getBytes(UTF_8));
-  public static final ByteString TE_TRAILERS = new ByteString(GrpcUtil.TE_TRAILERS.getBytes(UTF_8));
-  public static final ByteString USER_AGENT =
-      new ByteString(USER_AGENT_KEY.name().getBytes(UTF_8));
+  public static final AsciiString STATUS_OK = new AsciiString("200".getBytes(US_ASCII));
```

Comment:
Why not use `new AsciiString("200")` or `AsciiString.of("200")`? It seems simpler and would actually be more efficient from what I can tell.